### ISSUE TITLE
[vpc] Update nat instance IAM roles

### ISSUE THREAD

**Issue by limed** [Nubisproject/nubis-stacks#264]
We need to update the roles to allow the NSM package to install. Need the following 

``` json
        {
            "Resource": [
                "arn:aws:s3:::nsm-data"
            ],
            "Action": [
                "s3:ListBucket",
                "s3:GetBucketLocation"
            ],
            "Sid": "ListNSMFiles",
            "Effect": "Allow"
        },
        {
            "Resource": [
                "arn:aws:s3:::nsm-data/*"
            ],
            "Action": [
                "s3:GetObject"
            ],
            "Sid": "GetNSMFiles",
            "Effect": "Allow"
        }
```

**Comment by gozer** [comment]
Wait, what is that for? Installation is hapenning via the packer build or is this for something else? And how can this policy grant you access to an external bucket?

I guess I might be missing useful context here!

**Comment by limed** [comment]
You are probably right here, I was debugging something and wanted to add this before I forget but forgot to realize that I didn't need an instance to install it

**Comment by limed** [comment]
Fixed in PR #267
